pkg/driver/deployment_awslambda.go

```diff
@@ -88,13 +85,153 @@ func CleanAWSLambda(functions []*common.Function) {
 		wg.Wait()
 	}
 
-	log.Debugf("Deleted %d out of %d serverless.yml files", counter, len(functionGroups))
+	if counter != uint64(len(functionGroups)) {
+		log.Errorf("Deleted %d out of %d serverless.yml files", counter, len(functionGroups))
```

Comment:
This case would print both "deleted x out of y" and "deleted all" lines. I guess, it should be returning after that.